--- wxPython-src-2.8.12.1/wxPython/src/gtk/_core_wrap.cpp.orig	2011-07-14 04:25:06.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/_core_wrap.cpp	2013-11-03 14:55:41.362331369 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/_gdi_wrap.cpp.orig	2011-07-14 04:25:09.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/_gdi_wrap.cpp	2013-11-03 15:05:50.555639121 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/_windows_wrap.cpp.orig	2011-07-14 04:25:12.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/_windows_wrap.cpp	2013-11-03 15:12:02.712290262 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/_controls_wrap.cpp.orig	2011-07-14 04:25:16.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/_controls_wrap.cpp	2013-11-03 15:26:33.475586961 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/_misc_wrap.cpp.orig	2011-07-14 04:25:18.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/_misc_wrap.cpp	2013-11-03 15:38:27.618890331 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/calendar_wrap.cpp.orig	2011-07-14 04:25:20.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/calendar_wrap.cpp	2013-11-03 15:46:31.515536683 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/combo_wrap.cpp.orig	2011-07-14 04:25:22.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/combo_wrap.cpp	2013-11-03 15:55:47.475513353 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/grid_wrap.cpp.orig	2011-07-14 04:25:24.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/grid_wrap.cpp	2013-11-03 16:06:53.915485385 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/html_wrap.cpp.orig	2011-07-14 04:25:27.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/html_wrap.cpp	2013-11-03 16:27:42.525432993 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/media_wrap.cpp.orig	2011-07-14 04:25:28.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/media_wrap.cpp	2013-11-03 16:37:51.125407447 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/webkit_wrap.cpp.orig	2011-07-14 04:25:29.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/webkit_wrap.cpp	2013-11-03 16:48:08.388714936 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/wizard_wrap.cpp.orig	2011-07-14 04:25:31.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/wizard_wrap.cpp	2013-11-03 16:57:12.832025305 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/xrc_wrap.cpp.orig	2011-07-14 04:25:32.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/xrc_wrap.cpp	2013-11-03 17:04:54.445339321 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/richtext_wrap.cpp.orig	2011-07-14 04:25:35.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/richtext_wrap.cpp	2013-11-03 17:26:00.805286181 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/aui_wrap.cpp.orig	2011-07-14 04:25:38.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/aui_wrap.cpp	2013-11-03 17:36:52.331925504 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/animate_wrap.cpp.orig	2011-07-14 04:25:39.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/animate_wrap.cpp	2013-11-03 17:51:41.838554843 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/glcanvas_wrap.cpp.orig	2011-07-14 04:25:41.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/glcanvas_wrap.cpp	2013-11-03 18:13:51.981832351 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp.orig	2011-07-14 04:25:47.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/contrib/gizmos/gtk/gizmos_wrap.cpp	2013-11-03 18:14:56.971829612 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-2.8.12.1/wxPython/src/gtk/stc_wrap.cpp.orig	2011-07-14 04:25:43.000000000 +0200
+++ wxPython-src-2.8.12.1/wxPython/src/gtk/stc_wrap.cpp	2013-11-03 18:15:21.091828613 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-3.0.0.0/wxPython/src/gtk/dataview_wrap.cpp~	2013-12-16 14:51:45.000000000 +0100
+++ wxPython-src-3.0.0.0/wxPython/src/gtk/dataview_wrap.cpp	2014-01-15 12:42:47.698683686 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-3.0.0.0/wxPython/src/gtk/propgrid_wrap.cpp~	2013-12-16 14:52:01.000000000 +0100
+++ wxPython-src-3.0.0.0/wxPython/src/gtk/propgrid_wrap.cpp	2014-01-15 12:59:30.358303483 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
--- wxPython-src-3.0.0.0/wxPython/src/gtk/html2_wrap.cpp~	2013-12-28 10:28:56.000000000 +0100
+++ wxPython-src-3.0.0.0/wxPython/src/gtk/html2_wrap.cpp	2014-01-15 13:09:44.188126337 +0100
@@ -861,7 +861,7 @@
     Py_DECREF(old_str);
     Py_DECREF(value);
   } else {
-    PyErr_Format(PyExc_RuntimeError, mesg);
+    PyErr_Format(PyExc_RuntimeError, "%s", mesg);
   }
 }
 
