--- wxPython-4.1.0/buildtools/config.py.orig	2024-11-17 13:47:19.336769778 +0100
+++ wxPython-4.1.0/buildtools/config.py	2024-11-17 13:47:21.916755801 +0100
@@ -221,11 +221,6 @@ class Configuration(object):
 
             self.cflags = self.getWxConfigValue('--cxxflags')
             self.cflags = self.cflags.split()
-            if self.debug:
-                self.cflags.append('-ggdb')
-                self.cflags.append('-O0')
-            else:
-                self.cflags.append('-O3')
 
             lflags = self.getWxConfigValue('--libs')
             self.MONOLITHIC = (lflags.find("_xrc") == -1)
