--- wxPython-src-3.0.0.0/wxPython/config.py~	2013-12-16 14:39:50.000000000 +0100
+++ wxPython-src-3.0.0.0/wxPython/config.py	2014-01-15 09:19:01.973976105 +0100
@@ -1030,11 +1030,6 @@
 
     cflags = getWxConfigValue('--cxxflags')
     cflags = cflags.split()
-    if debug:
-        cflags.append('-ggdb')
-        cflags.append('-O0')
-    else:
-        cflags.append('-O3')
 
     lflags = getWxConfigValue('--libs')
     MONOLITHIC = (lflags.find("_xrc") == -1)
