--- wxPython-2.3.2.1/setup.py.wiget	Wed Jun  5 05:06:40 2002
+++ wxPython-2.3.2.1/setup.py	Wed Jun  5 05:07:12 2002
@@ -32,7 +32,7 @@
 BUILD_OGL = 1      # If true, build the contrib/ogl extension module
 BUILD_STC = 1      # If true, build the contrib/stc extension module
 BUILD_XRC = 1      # XML based resource system
-BUILD_GIZMOS = 1   # Build a module for the gizmos contrib library
+BUILD_GIZMOS = 0   # Build a module for the gizmos contrib library
 BUILD_DLLWIDGET = 1# Build a module for the gizmos contrib library
 
 BUILD_IEWIN = 0    # Internet Explorer wrapper (experimental)
